website/docs/d/search_indexes.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: search indexes"
+sidebar_current: "docs-mongodbatlas-search-indexes"
+description: |-
+Describes a Search Indexes.
+---
+
+# mongodbatlas_search_indexes
+
+`mongodbatlas_search_indexes` describe all Projects. This represents projects that have been created.
```

Comment:
This is likely a copy and paste mistake :)

Reply:
yes! my bad